bin/transync

```diff
@@ -59,3 +59,21 @@ if (codingStyle.insert_final_newline) {
 fs.writeFile(program.to, newContent, 'utf8', function () {
   echo(`Version ‘${program.to}’ correctly synchronized with base version ‘${program.from}’.`)
 })
+
+/**
+ * sync translation dictionaries from `from` to `to`.
+ *
+ * this supports nested dictionaries and will remove keys
+ * from the to dictionary that is not in the from dictionary.
+ */
+function syncDictionaries(from, to) {
```

Comment:
I think we could simply use an existing deep assign like https://www.npmjs.com/package/object-assign-deep.